jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Jar.java

```diff
@@ -67,11 +78,71 @@
 
   @CommandLine.Option(
       names = "--from",
-      paramLabel = "<from>",
+      paramLabel = "<base-image>",
       description = "The base image to use.")
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private String from;
 
+  @CommandLine.Option(
+      names = "--jvm-flags",
+      paramLabel = "<jvm-flags>",
+      split = ",",
+      description = "JVM arguments.")
```

Comment:
I think it's worth adding a comma-separated usage example. `-Dmy.property=value,-Xshare:off`.

Reply:
Good idea!